homeassistant/components/sensor/sql.py

```diff
@@ -0,0 +1,129 @@
+"""
+Sensor from an SQL Query.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.sql/
+"""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    CONF_NAME, CONF_UNIT_OF_MEASUREMENT, CONF_VALUE_TEMPLATE,
+    STATE_UNKNOWN)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+REQUIREMENTS = ['sqlalchemy==1.2.2']
+
+DEFAULT_NAME = 'Sql_sensor'
+
+CONF_QUERIES = 'queries'
+CONF_QUERY = 'query'
+CONF_COLUMN_NAME = 'column'
+CONF_DB_URL = 'db_url'
```

Comment:
`CONF_DB_URL` is already defined by the `recorder` component, could be imported from there.